apps/web/src/app/api/trpc/[...trpc]/route.ts

```diff
@@ -0,0 +1,13 @@
+import { fetchRequestHandler } from "@trpc/server/adapters/fetch"
+import { appRouter, createContext } from "@dotkomonline/gateway-trpc"
+
+const handler = (req: Request) => {
+  return fetchRequestHandler({
+    endpoint: "/api/trpc",
+    req,
+    router: appRouter,
+    createContext,
+  })
+}
```

Comment:
if we deleted /api/trpc where does this call?

Reply:
Itself, it was just moved to app/api instead of pages/api